rest-api/dao/physical_measurements_dao.py

```diff
@@ -303,7 +303,8 @@ def _update_participant_summary(self, session, obj):
         participant_summary.physicalMeasurementsStatus = PhysicalMeasurementsStatus.CANCELLED
         participant_summary.physicalMeasurementsTime = None
 
-    if participant_summary.physicalMeasurementsStatus != PhysicalMeasurementsStatus.CANCELLED:
+    if participant_summary.physicalMeasurementsStatus != PhysicalMeasurementsStatus.CANCELLED or \
```

Comment:
rebase? I thought you already made this change?

Reply:
I should have started a new PR. I'll close this out.